Cargo.toml

```diff
@@ -12,7 +12,7 @@ edition = "2018"
 
 [dependencies]
 failure = { version = "0.1.6", features = ["derive"] }
-flexi_logger = "0.14.4"
+flexi_logger = "0.14.5"
```

Comment:
Woahhhh this is a huge change that should have more documentation. (Actually though is there any reason?)